[GitHub]_ABCDs_Detector.ipynb

```diff
@@ -182,47 +171,49 @@
         "   * The **Run ABCDs Detector** step produces the video analysis.\n",
         "\n",
         "1. For questions, please reach out to: abcds-detector@google.com"
-      ],
-      "metadata": {
-        "id": "XHqQskKjxtEk"
-      }
+      ]
     },
     {
       "cell_type": "markdown",
-      "source": [
-        "# Install ABCD Detector Code"
-      ],
       "metadata": {
         "id": "m-VVH3V36y8y"
-      }
+      },
+      "source": [
+        "# Install ABCD Detector Code"
+      ]
     },
     {
       "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "id": "3KlVPryttrZr"
+      },
+      "outputs": [],
       "source": [
         "!git clone -b paul_changes https://github.com/google-marketing-solutions/abcds-detector\n",
```

Comment:
Remove -b paul_changes, this was an artifact left over from testing the release, it was not an issue while the branch and head were in sync, however now that additional changes are in, this should be removed.